lib/hawkular/alerts/alerts_api.rb

```diff
@@ -20,6 +20,13 @@ def initialize(entrypoint = 'http://localhost:8080/hawkular/alerts', credentials
       super(entrypoint, credentials, options)
     end
 
+    # Return version and status information for the used version of Hawkular-Alerting
+    # @return [Hash{String=>String}]
+    #         ('Implementation-Version', 'Built-From-Git-SHA1', 'status')
+    def fetch_version_and_status
```

Comment:
Sorry, I am not agree here.
def fetch_version_and_status is not new code, it is to complete the same method used in other subclient to another subclient.
A change on the method will imply a bump of the version, and for this minor version I don't think is worth to do it.
So, a review of the PR (and its discussion) should go in a following PR not in this.